app/lib/seeder.rb

```diff
@@ -9,6 +10,24 @@ class Seeder; end
 class Seeder
   include ActiveSupport::Testing::TimeHelpers
 
+  def run
+    return if Rails.env.production?
+
+    self.class.load_fraud_indicators
+
+    Flipper.enable(:eitc)
```

Comment:
any other flippers we should enable or should we delete them from different environments?